python-sdk/src/astro/sql/operators/append.py

```diff
@@ -59,8 +62,43 @@ def execute(self, context: Context) -> BaseTable:  # skipcq: PYL-W0613
             target_table=self.target_table,
             source_to_target_columns_map=self.columns,
         )
+
         return self.target_table
 
+    def get_openlineage_facets(self) -> OpenLineageFacets:
+        input_dataset: list[OpenlineageDataset] = [
+            OpenlineageDataset(
+                namespace=self.source_table.openlineage_dataset_namespace(),
+                name=self.source_table.openlineage_dataset_name(),
+                facets={
+                    "table_name": self.source_table.name,
+                    "row_affected": 0,  # FixMe
+                    "columns": self.columns,
+                    "metadata": self.target_table.metadata,
+                },
+            )
+        ]
+
+        output_dataset: list[OpenlineageDataset] = [
+            OpenlineageDataset(
+                namespace=self.target_table.openlineage_dataset_namespace(),
+                name=self.target_table.openlineage_dataset_name(),
+                facets={
```

Comment:
@pankajastro This has to be an object from class Basefacet. You might need to add a custom facet class like https://github.com/astronomer/astro-sdk/blob/main/python-sdk/src/astro/sql/operators/load_file.py#L205

Reply:
Added let me know how it looks now

Reply:
LGTM